src/reference/workflow_nodes.md

```diff
@@ -1,124 +1,62 @@
----
-description: "Find detailed reference information on Caido workflow nodes including JavaScript, Shell, If/Else, and other automation components."
----
-
```

Comment:
Can we add a description back in for this page? The purpose of these is to have better search result descriptions on Google.